guides/common/modules/proc_installing-postgresql.adoc

```diff
@@ -27,6 +27,14 @@ rh-postgresql12-postgresql-evr
 . Edit the `/var/opt/rh/rh-postgresql12/lib/pgsql/data/postgresql.conf` file:
 +
 [options="nowrap" subs="verbatim,quotes"]
+
```

Comment:
@ianf77 , when you have a sec, could you give this a peer review glance, please?